validation/observability/stackstate_server_test.go

```diff
@@ -103,7 +103,7 @@ func (sss *StackStateServerTestSuite) SetupSuite() {
 	}
 }
 
-func (sss *StackStateServerTestSuite) TestInstallStackState() {
+func (sss *StackStateServerTestSuite) TestInstallStackStateServerNonHAValues() {
```

Comment:
since the suite already has StackState in the name, maybe just `TestInstallServerNonHAValues`?

Reply:
Renamed